addons/gaea/resources/gaea_value.gd

```
@@ -230,7 +232,7 @@ static func get_default_slot_icon(type: Type) -> Texture2D:
 			return load("uid://yo87adchyr3w")
 		Type.MAP:
 			return load("uid://d2rmsal7c6sdi")
-	return load("uid://dqob6v3dudlri")
+	return null
```

Now we will get silent errors if the slot icon is missing, maybe add a push_warning before returning the default icon ?